deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -698,37 +705,61 @@ groups:
           type: ceph_default
           oid: 1.3.6.1.4.1.50495.1.2.1.8.4
         annotations:
-          summary: Host filesystem freespace is getting low
+          summary: Host filesystem free space is low
           description: >
             Mountpoint {{ $labels.mountpoint }} on {{ $labels.nodename }}
-            will be full in less than 5 days assuming the average fill-up
-            rate of the past 48 hours.
+            will be full in less than 5 days based on the 48 hour trailing
+            fill rate.
 
       - alert: CephNodeInconsistentMTU
         expr: node_network_mtu_bytes{device!="lo"} * (node_network_up{device!="lo"} > 0) != on() group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}))
         labels:
           severity: warning
           type: ceph_default
         annotations:
-          summary: MTU settings across Ceph hosts are inconsistent
+          summary: MTU settings across hosts are inconsistent
           description: >
             Node {{ $labels.instance }} has a different MTU size ({{ $value }})
             than the median value on device {{ $labels.device }}.
 
   - name: pools
     rules:
+      - alert: CephPoolGrowthWarning
```

Comment:
Did you define this alert, or it's picked up from the ceph repo?

Reply:
The latter, I noticed that it was missing from Rook.

Reply:
It's definitely been a few months since we picked it up, sounds good to add it